### PR TITLE
Newsletter-70: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

**Comment by bittomhan** [comment]
I really do not know what happend so that the `make production` can not link to the page.

Screenshot 2024-09-17 at 10 46 54 PM

May be the `en` and `es` version is OK.

Screenshot 2024-09-17 at 10 46 45 PM

So should I just pull request the linking page `Bitcoin Optech Schnorr Taproot Workshop` first? Or what should I do to fix this?

File: _posts/zh/2019-10-29-schnorr-taproot-workshop.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+title: Bitcoin Optech Schnorr Taproot Workshop
+permalink: /zh/schorr-taproot-workshop/
+name: 2019-10-29-schorr-taproot-workshop-zh
```
Comment (bitschmidty) [inline _posts/zh/2019-10-29-schnorr-taproot-workshop.md:4]:
```suggestion
name: 2019-10-29-schnorr-taproot-workshop-zh
```

File: _posts/zh/2019-10-29-schnorr-taproot-workshop.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+type: posts
+layout: post
+lang: zh
+slug: 2019-10-21-schorr-taproot-workshop-zh
```
Comment (bitschmidty) [inline _posts/zh/2019-10-29-schnorr-taproot-workshop.md:8]:
```suggestion
slug: 2019-10-21-schnorr-taproot-workshop-zh
```

File: _posts/zh/2019-10-29-schnorr-taproot-workshop.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
+---
+title: Bitcoin Optech Schnorr Taproot Workshop
+permalink: /zh/schorr-taproot-workshop/
```
Comment (bitschmidty) [inline _posts/zh/2019-10-29-schnorr-taproot-workshop.md:3]:
```suggestion
permalink: /zh/schnorr-taproot-workshop/
```

**Review by bitschmidty — COMMENTED** [review]
We screwed up on the URL/permalink here in the original and other translations

schorr vs schnorr

So these suggestions should be the correct ones. But first we need to correct other versions. Ill open a PR now

File: _posts/zh/newsletters/2019-10-30-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+[pr14451]: /zh/newsletters/2018/10/30/#bitcoin-core-14451
+[heartbleed]: https://bitcoin.org/en/alert/2014-04-11-heartbleed
+[ber]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-July/009697.html
+[pr15681]: /zh/newsletters/2019/07/24/#bitcoin-core-15681
+[opt_simplified]: /zh/newsletters/2018/11/27/#simplified-fee-bumping-for-ln
+[anchor thread]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-October/002249.html
+[halseth carve-out]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-October/002240.html
+[daftuar duo]: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/Mempool-and-mining
+[rubin justification]: http://www.erisian.com.au/bitcoin-core-dev/log-2019-10-24.html#l-660
+[taproot workshop]: /zh/schorr-taproot-workshop/
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2019-10-30-newsletter.md:73]:
```suggestion
[taproot workshop]: /zh/schnorr-taproot-workshop/
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Comment by bittomhan** [comment]
@bitschmidty Thank you for your efforts and I learned what is hugo conversion.
But it seemed still did not slove the problem yet?

File: _posts/zh/2019-10-29-schnorr-taproot-workshop.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+type: posts
+layout: post
+lang: zh
+slug: 2019-10-21-schnorr-taproot-workshop-zh
+
+excerpt: >
+  一门自学课程，用于学习 schnorr/taproot 软分叉提案。
+
+auto_id: false
+------
```
Comment (bitschmidty) [inline _posts/zh/2019-10-29-schnorr-taproot-workshop.md:14]:
```suggestion
---
```

**Review by bitschmidty — COMMENTED** [review]
Took a bit of playing around but I think this is the issue

**Comment by bittomhan** [comment]
@bitschmidty 
It seemed like something almost there since it shows the link to `en` version has promblem on the `zh` file.
Screenshot 2024-09-21 at 9 23 38 PM

**Comment by bitschmidty** [comment]
@bittomhan can you rebase on master which has the changes from https://github.com/bitcoinops/bitcoinops.github.io/pull/1902 regarding these pages?

**Comment by bittomhan** [comment]
> @bittomhan can you rebase on master which has the changes from #1902 regarding these pages?

It works! Thank you very much!

**Pull request merged**